bridge/ApiRequestHandler.cpp

```diff
@@ -381,6 +381,11 @@ ApiRequestHandler::ApiRequestHandler(bridge::MixerManager& mixerManager, transpo
       _legacyApiRequestHandler(std::make_unique<LegacyApiRequestHandler>(mixerManager, sslDtls))
 #endif
 {
+#if ENABLE_LEGACY_API
+    logger::logAlways("Legacy colibri API enabled", "ApiRequestHandler");
+#else
+    logger::logAlways("Legacy API disabled", "ApiRequestHandler");
+#endif
```

Comment:
Could you remove this? I understand that you needed it at first, but legacy api is for Symphony internal builds only.